homeassistant/components/reolink/host.py

```diff
@@ -97,7 +98,8 @@ async def async_init(self) -> None:
                 f"'{self._api.user_level}', only admin users can change camera settings"
             )
 
-        self._onvif_supported = self._api.supported(None, "ONVIF")
+        self._onvif_push_supported = self._api.supported(None, "ONVIF")
+        self._onvif_long_poll_supported = self._api.supported(None, "ONVIF")
```

Comment:
You're doing the same call twice, is there significance of calling it twice? Can we store it in a local object first?

Reply:
done